source/server/server.cc

```diff
@@ -529,8 +529,11 @@ void InstanceImpl::initialize(Network::Address::InstanceConstSharedPtr local_add
                                      ? ReusePortDefault::True
                                      : ReusePortDefault::False;
   } else {
-    // This is needed so that we don't read the value until runtime is fully initialized.
-    enable_reuse_port_default_ = ReusePortDefault::Runtime;
+    #if defined(WIN32) || defined(__APPLE__)
```

Comment:
It is already disabled by here

https://github.com/envoyproxy/envoy/blob/158535979a833d076c44c701f1cdf5406187d3ba/source/server/listener_impl.cc#L920-L931

So I guess we needn't disable it here again

Reply:
OK,thanks.